psi4/src/psi4/dct/dct_compute_RHF.cc

```diff
@@ -129,7 +130,8 @@ void DCTSolver::run_simult_dct_RHF() {
         "\t*---------------------------------------------------------------------------------*\n");
     auto tmp = std::make_shared<Matrix>("temp", nirrep_, nsopi_, nsopi_);
     // Set up the DIIS manager
-    DIISManager diisManager(maxdiis_, "DCT DIIS vectors");
+    py::object diis_file = py::module_::import("psi4").attr("driver").attr("scf_proc").attr("diis");
+    py::object diis_manager = diis_file.attr("DIIS")(maxdiis_, "DCT DIIS vectors");
```

Comment:
Is there any way to write a a wrapper or something so that this can be hidden? It's not very friendly.

Reply:
Perhaps the old C++ DIISManager can to be changed to do all of this.

Reply:
Francesco asked me the same thing. With two votes in favor for exploring that option, I'll give it a shot.